aspects/aspects.go

```diff
@@ -83,7 +83,7 @@ func notFoundErrorFrom(a *Aspect, op, request, errMsg string) *NotFoundError {
 		BundleName: a.bundle.Name,
 		Aspect:     a.Name,
 		Operation:  op,
-		Request:    request,
+		Request:    fmt.Sprintf("%q", request),
```

Comment:
why this change? it's a bit odd tbh

Reply:
It's because aspectstate now also uses this error with potentially more than one request and quoting that looked strange. But you're right, in hindsight this is very hacky. I'll change the field to acomodate it